SalesforceCommerceCloud/pwa-kit
# Fix Page Designer ImageWithText Link component

# Description



The initial simple solution is to keep using Chakra UI Link component in the PD ImageWithText Link component.

A more elaborated solution uses `isAbsoluteURL` util to conditionally use Chakra UI Link component for absolute URLs and our custom react-router Link component for relative URLs.

TODO: 
 - [x] Remove the example page viewer page and its route in routes.jsx.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Updated ImageWithText Link component.

# How to Test-Drive This PR

- Start `template-retail-react-app` and load http://localhost:3000/page-viewer/get-ready-for-summer.
- Click on the images links, and verify all links use hard navigation except the Kids link which does a soft navigation to a relative URL. 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
